src/project_file_walker.rs

```diff
@@ -34,7 +37,10 @@ impl WalkParallelIterator {
                     let sender = sender.clone();
                     move |entry| {
                         let entry = match entry {
-                            Err(_) => return WalkState::Continue,
+                            Err(err) => {
+                                err_message!("{err}");
```

Comment:
This is where the file not found error is surfaced from `WalkParallel` (for both directory and file path command-line arguments)